catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogPropertiesMetadata.java

```diff
@@ -53,6 +53,10 @@ public class HadoopCatalogPropertiesMetadata extends BaseCatalogPropertiesMetada
    */
   public static final String DEFAULT_FS_PROVIDER = "default-filesystem-provider";
 
+  static final String FILESYSTEM_CONNECTION_TIMEOUT_SECONDS =
+      "filesystem-connection-timeout-seconds";
```

Comment:
The configuration name is too long, can you make it shorter.

Reply:
How about "filesystem-conn-timeout-sec"?

Reply:
I take it.